ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml

```diff
@@ -62,3 +62,4 @@
     - nasm                        # OpenJ9
     - Rust                        # IcedTea-Web
     - IcedTea-Web                 # For Jenkins webstart
+    - Build                       # Role to build JDK8
```

Comment:
Only JDK 8?

Reply:
For now- I can look into generalising that role (a lot like I did in #955) so we can build multiple, and then give the option to choose which JDK to build in the script. Currently this is just meant to test if the playbook installs all the dependencies required for 8

Reply:
OK, we can update that in a later PR